### PR TITLE
feat(swapper): add getByPair

### PR DESCRIPTION
**Pull request by DaoDev44**
- Add `isSupportedAssets` to SwapperManager.
- This is needed to chose swappers by supported assets instead of hard coding the swapper to use.

Closes [Add logic to get swapper by pair](https://github.com/shapeshift/lib/issues/523) #523

### CODE REVIEW

File: packages/swapper/src/swappers/zrx/getZrxSupportedAssets/getZrxSupportedAssets.test.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+describe('getZrxSupportedAssets', () => {
+  it('is true', () => {
+    expect('yes').toEqual('yes')
```
Comment (chxash) [inline packages/swapper/src/swappers/zrx/getZrxSupportedAssets/getZrxSupportedAssets.test.ts:3]: Not sure about this, maybe a leftover ? :) even if its very basic we can unit test `getSupportedAssets` and `isSupportedAssets` methods

**Review by chxash — APPROVED** [review]
LGTM !

File: packages/swapper/src/swappers/zrx/getZrxSupportedAssets/getZrxSupportedAssets.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { CAIP19 } from '@shapeshiftoss/caip'
+
+export function getZrxSupportedAssets(): CAIP19[] {
```
Comment (elmutt) [inline packages/swapper/src/swappers/zrx/getZrxSupportedAssets/getZrxSupportedAssets.ts:3]: IMO this is fine for now and doesnt add much tech debt while its being decided

**Review by elmutt — APPROVED** [review]

File: packages/swapper/src/api.ts [hunk 3]
```diff
@@ -76,4 +85,9 @@ export interface Swapper {
    * Get max swap balance (minus fees) for sell asset
    */
   getSendMaxAmount(args: SendMaxAmountInput): Promise<string>
+
+  /**
+   * Get a boolean if swapper supports an asset
+   */
+  isSupportedAssets(args: SupportedAssetInput): boolean
```
Comment (cjthompson) [inline packages/swapper/src/api.ts:92]: What does `supported` mean?

We talked about the possibility of an asset being available on only one side of a trade, like being a "sell" or "buy" asset but not both.

**Review by cjthompson — CHANGES_REQUESTED** [review]
I'd like to discuss `isSupportedAssets`

File: packages/swapper/src/manager/SwapperManager.ts [hunk 4]
```diff
@@ -61,25 +60,43 @@ export class SwapperManager {
     return this
   }
 
+  /**
+   *
+   * @param quoteParams type {GetQuoteInput}
+   * @returns {SwapperType}
+   */
```
Comment (cjthompson) [inline packages/swapper/src/manager/SwapperManager.ts:67]: Remove?

File: packages/swapper/src/manager/SwapperManager.ts [hunk 5]
```diff
@@ -61,25 +60,43 @@ export class SwapperManager {
...
   async getBestSwapper(quoteParams: GetQuoteInput): Promise<SwapperType> {
     quoteParams // noop to shut up linter
     return SwapperType.Zrx // TODO: implement getBestSwapper
   }
 
-  getSupportedBuyAssetsFromSellId(args: BuyAssetBySellIdInput) {
+  /**
+   *
```
Comment (cjthompson) [inline packages/swapper/src/manager/SwapperManager.ts:74]: Please add a description of the function here

**Review by cjthompson — APPROVED** [review]
I'd like to discuss `isSupportedAssets`

**Pull request merged**